airbyte-integrations/connectors/source-recurly/source_recurly/streams.py

```diff
@@ -17,6 +18,7 @@
 
 BEGIN_TIME_PARAM = "begin_time"
 END_TIME_PARAM = "end_time"
+RATE_LIMIT_SLEEP_IN_SECS = 0.16
```

Comment:
is this to make 1000 req/min?

Reply:
The goal is to make it less than 2000 calls per 5 minutes. With this value, we make it about 1875 per 5 minutes.